DEVDOCS-2227: Add Section on Global Regions (#713)
bigcommerce/dev-docs: docs/api-docs/storefront/widgets/widgets-overview.md

## Regions

> [Regions](https://developer.bigcommerce.com/api-reference/storefront/widgets-api/regions/getcontentregions) are specific locations in the Stencil theme files where you can place a widget. You can add a region at the file level using the format `{{{region name="..."}}}` and name the region however you like, but it is best practice to give it a name that is descriptive of the location and function. A theme file can have as many regions as you want, with more than one widget assigned to the region and the [Placements](https://developer.bigcommerce.com/api-docs/store-management/widgets/overview#placements) `sort_order` controlling how the widgets appear on the storefront.

Reviewer: @ektravel, do we have an example of the best practice name formatting we mention here? It might be in the screenshot, but I can't see it in this view. Let me know.

Author: @uwikaiddi The screenshot of home.html highlights three different regions. I can add another example if needed but I think the screenshot does a good job of illustrating the suggested naming convention.

Reviewer: Okay, thanks @ektravel

![Content Region](//s3.amazonaws.com/user-content.stoplight.io/6012/1551970794664 "Content Region")

![Content Region](//s3.amazonaws.com/user-content.stoplight.io/6012/1551970767918 "Content Region")

Most themes in the BigCommerce marketplace come with predefined regions. It is best to use those first. By editing the theme and adding theme regions, updates will need to be managed manually.

### Global regions

> Global regions are special regions that can be used to place and manage content sitewide. Widgets placed in a global region will appear on every page of the storefront where that region exists. This makes global regions useful for displaying high-priority information, such as special promotion advertising banners, across multiple pages of the storefront.

Reviewer 2: ```suggestion
Global regions are special regions you can use to place and manage content sitewide. Widgets placed in a global region will appear on every page of the storefront where that region exists. This makes global regions useful for displaying high-priority information, such as special promotion advertising banners, across multiple storefront pages.
```
